interview-questions.md

```diff
@@ -8,50 +8,123 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 INSTRUCTOR EXAMPLE: What is a conditional statement?
 
-Your answer:
+Your answer: 
+
+ conditional statement is based off a mathmatical equation such as a for loop or a boolean
+
+Researched answer: 
+   A way for computers to make decisions based on conditions. 
+
+Source: 
+    https://technovationchallenge.org/curriculum/coding-7/#:~:text=Conditionals%20Statements%20%2D%20A%20way%20for,is%20false%2C%20do%20another%20thing.
+
+
 
-Researched answer:
 
 1. What is git? What is the difference between git and Github?
 
-Your answer:
+Your answer: 
+   
+   Let's start with Github. Github is what I call a cloud- based family tree, Its where my cohorts and I go when we need to build a project together or individually. Git is where I store my personal source code on my personal computer it also helps me keep track of the changed I have made.
 
 Researched answer:
 
+  Git is software for tracking changes in any set of files, usually used for coordinating work among programmers collaboratively developing source code during software development. The difference between Git and GitHub: Git is a widely used version control system that lets you manage and keep track of your code. GitHub is a cloud-based hosting service that lets you manage your Git repositories. With GitHub, you can back up your personal files, share your code, and collaborate with others.
+ 
+ Source: 
+   https://en.wikipedia.org/wiki/Git
+   https://www.codecademy.com/article/what-is-git-and-github-desktop
+
+
 2. Which JavaScript operators will return a Boolean value?
 
-Your answer:
+Your answer: 
+
+   The javaScript operatprs that will return a Boolean value are && (andAnd). If both sides of the condition is satisfied you will use &&. || (or) You use this operator when one or the other condition is true. For the Big Bang (!) you usually use this when the conditions are negated.
+
+Researched answer: 
```

Comment:
What are the three categories of operators that give a Boolean value? You have listed the logical operators. Two more categories to go.